package-lock.json

```diff
@@ -0,0 +1,15982 @@
+{
```

Comment:
I don't believe this file should be included as part of the change?

Reply:
Maybe #575's author is using npm other than yarn?
Should I include a line to exclude it to .gitignore for future comers?